pandas/tests/frame/test_indexing.py

```diff
@@ -2626,6 +2626,23 @@ def test_index_namedtuple(self):
         result = df.loc[IndexType("foo", "bar")]["A"]
         assert result == 1
 
+    def test_index_single_double_tuples(self):
```

Comment:
can parameterize this (e.g. pass in what you have as tuple_1, tuple_2)

Reply:
thanks for the feedback @jreback - now using parameterize